packages/widget-chat/src/components/activity-title-bar/index.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {findRenderedDOMComponentWithClass, renderIntoDocument} from 'react-addons-test-utils';
+
+import ActivityTitleBar from '.';
+
+let user;
+let component;
+
+describe(`ActivityTitleBar component`, () => {
```

Comment:
Is this a title bar for a single activity or for a conversation's list of activities?